sdk/search/azure-search/src/samples/java/com/azure/search/CreateSkillsetExample.java

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search;
+
+import com.azure.core.util.Configuration;
+import com.azure.search.models.InputFieldMappingEntry;
+import com.azure.search.models.OcrSkill;
+import com.azure.search.models.OutputFieldMappingEntry;
+import com.azure.search.models.Skill;
+import com.azure.search.models.Skillset;
+import com.azure.search.models.WebApiSkill;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+
+public class CreateSkillsetExample {
+
+    /**
+     * From the Azure portal, get your Azure Cognitive Search service URL and API key,
+     * and set the values of these environment variables:
+     */
+    private static final String ENDPOINT = Configuration.getGlobalConfiguration().get("AZURE_COGNITIVE_SEARCH_ENDPOINT");
+    private static final String ADMIN_KEY = Configuration.getGlobalConfiguration().get("AZURE_COGNITIVE_SEARCH_ADMIN_KEY");
+
+    public static void main(String[] args) {
+        SearchServiceClient searchServiceClient = new SearchServiceClientBuilder()
+            .endpoint(ENDPOINT)
+            .credential(new SearchApiKeyCredential(ADMIN_KEY))
+            .buildClient();
+
+        createOcrSkillset(searchServiceClient);
+        createCustomSkillset(searchServiceClient);
+    }
+
+    private static void createOcrSkillset(SearchServiceClient searchServiceClient) {
+        List<InputFieldMappingEntry> inputs = Arrays.asList(
+            new InputFieldMappingEntry()
+                .setName("url")
```

Comment:
Curious how this was selected to use the url input.  This is something that we haven't documented and frankly don't want to have to support forever, is really just a feature that our PMs can use when they need to.  Is it possible to update this to use the image input instead?  Would also be a chance to demonstrate changing the skill context if you wanted. https://docs.microsoft.com/en-us/azure/search/cognitive-search-skill-ocr

Reply:
No particular reason, just taken from one of the unit tests and exposed as a sample. Sample is update with image input